packages/frontend-shared/src/components/ResultCounts.vue

```diff
@@ -36,33 +41,50 @@ const { t } = useI18n()
 
 const props = defineProps<ResultCountsProps>()
 
+const ORDER_MAP = {
+  'SKIPPED': {
+    value: props.totalSkipped,
+    class: 'icon-dark-gray-400',
+    icon: SkippedIcon,
+    name: t('runs.results.skipped'),
+  },
+  'PENDING': {
+    value: props.totalPending,
+    class: 'icon-dark-gray-400 icon-light-white',
+    icon: PendingIcon,
+    name: t('runs.results.pending'),
+  },
+  'PASSED': {
+    value: props.totalPassed,
+    class: 'icon-dark-jade-400',
+    icon: PassedIcon,
+    name: t('runs.results.passed'),
+  },
+  'FAILED': {
+    value: props.totalFailed,
+    class: 'icon-dark-red-400',
+    icon: FailedIcon,
+    name: t('runs.results.failed'),
+  },
+}
+
+interface status {
+  value: number|string|null
+  class: string
+  icon: FunctionalComponent<SVGAttributes, {}>
+  name: string
+}
+
 const results = computed(() => {
-  return [
-    {
-      value: props.totalSkipped,
-      class: 'icon-dark-gray-400',
-      icon: SkippedIcon,
-      name: t('runs.results.skipped'),
-    },
-    {
-      value: props.totalPending,
-      class: 'icon-dark-gray-400 icon-light-white',
-      icon: PendingIcon,
-      name: t('runs.results.pending'),
-    },
-    {
-      value: props.totalPassed,
-      class: 'icon-dark-jade-400',
-      icon: PassedIcon,
-      name: t('runs.results.passed'),
-    },
-    {
-      value: props.totalFailed,
-      class: 'icon-dark-red-400',
-      icon: FailedIcon,
-      name: t('runs.results.failed'),
-    },
-  ]
+  const acc: status[] = []
+
+  if (props.order) {
```

Comment:
Should the order even be configurable? The results this component displays should be consistent across features, otherwise it becomes confusing.

Reply:
This is an attempt to make the `ResultCounts.vue` component more generic. In its current implementation, the status can only appear in a certain order i.e. skipped, pending, passed, failed. You can see this generic component in the Runs tab for the app.
Based on the design requirements for this issue, the order of these "statuses" is different. Hence, instead of building a whole new component we can utilize `ResultCounts.vue` and just pass in a different order incase in the future the order of "statuses" is different.

As for the component being consistent, I agree with that. I don't see why the "status" order should differ across pages. However this has been added to meet the design requirements

Reply:
I would sync back with design regarding this, this result of this list (in my mind) should be consistent regardless of if you're looking at the runs page or the debug page